src/main/kotlin/de/danielmaile/mpp/item/Armors.kt

```diff
@@ -0,0 +1,72 @@
+/*
+ * This file is part of MPP.
+ * Copyright (c) 2023 by it's authors. All rights reserved.
+ * MPP is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * MPP is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with MPP.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package de.danielmaile.mpp.item
+
+import de.danielmaile.mpp.item.recipe.recipes.ArmorRecipe
+import org.bukkit.Material
+import org.bukkit.inventory.EquipmentSlot
+import org.bukkit.inventory.Recipe
+import org.bukkit.inventory.meta.ItemMeta
+import org.bukkit.inventory.meta.LeatherArmorMeta
+
+enum class Armors(
+    val armorMaterial: ArmorMaterial,
+    val slot: EquipmentSlot // Should we use this or create an enum like before
```

Comment:
We should decide what to do and remove the comment.

Reply:
What would you prefer to do? both ways are correct in fact, it's just a way to identify the armor piece.

Reply:
I would use an enum. This prevents using EquipmentSlot.HAND which would make no sense.